form.yml

```diff
@@ -0,0 +1,326 @@
+    $default: true
+    $help: Install all required packages to install SAP Netweaver
+  
+  virtual_addresses:
+    $name: Virtual Addresses
+    $type: edit-group
+    $prototype:
+      $key:
+          $type: text
+          $name: IP Address
+      $type: text
+      $name: Hostname
+
+  sidadm_user_checkbox:
+    $name: Create the SAP sidadm user and group before the installation
+    $type: boolean
+    $default: false      
+  sidadm_user:
+    $name: SAP User/Group
+    $visibleIf: .sidadm_user_checkbox == true
+    $type: group
+    $help: If this entry exists the user and group will be created before the installation, not otherwise
+    uid:
+      $name: User ID
+      $type: text
+    gid:
+      $name: Group ID
+      $type: text
+
+  sid_adm_password_checkbox:
+    $name: Provide SAP <sid>adm user password
+    $type: boolean
+    $default: false
+    $help: <sid>adm password is optional, master password will be used as default, if the value is not defined
+  sid_adm_password:
+    $name: SAP <sid>adm user password
+    $visibleIf: .sid_adm_password_checkbox == true
+    $type: password
+    $help: The password of the SAP <sid>adm user. If not set master password will be used as default
+    $optional: true
+  
+  sap_adm_password_checkbox:
+    $name: Provide SAP <sap>adm user password
+    $type: boolean
+    $default: false
+    $help: <sap>adm password is optional, master password will be used as default, if the value is not defined
+  sap_adm_password:
+    $name: SAP admin <sap>adm user password
+    $visibleIf: .sap_adm_password_checkbox == true
+    $type: password
+    $help: The password of the SAP <sap>adm user. If not set master password will be used as default
+    $optional: true
+  
+  master_password:
+    $name: Master password
+    $type: password
+    $help: Master password is used for all the SAP users that are created
+    $optional: false
+
+  clean_nfs:
+    $name: Clean NFS share content
+    $type: boolean
+    $default: true
+    $help: Clean /sapmnt/{sid} and /usr/sap/{sid}/SYS content. It will only work if ASCS node is defined.
+    $optional: false
+
+  sapmnt_inst_media:
+    $name: NFS share containing Netweaver SID folder
+    $type: text
+    $help: NFS share with the SID folder
+    $optional: false
+
+  use_swpm_sar_file:
+    $name: Provide SWPM installer sar archive for Netweaver installation 
+    $type: boolean
+    $default: false
+    $help: Mark this option if you want to use swpm sar archive file for the installation instead of already extracted SWPM installer
+  swpm_sar_file:
+    $name: Path to SWPM archive
+    $visibleIf: .use_swpm_sar_file == true
+    $type: text
+    $help: The path to swpm installer media archive in SAR format. You also need to provide the sapcar executable path for extraction
+    $optional: true
+  nw_extract_dir:
+    $name: Path to extract the Netweaver installation media
+    $visibleIf: .use_swpm_sar_file == true
+    $type: text
+    $default: /sapmedia/NW
```

Comment:
Change the default to something else. We use `sapmedia_extract/NW` in the terraform project